toolchain/check/import.cpp

```diff
@@ -258,6 +258,38 @@ static auto CopyAncestorNameScopesFromImportIR(
   return scope_cursor;
 }
 
+static auto LoadImportForOwningFunction(Context& context,
+                                        const SemIR::File& import_sem_ir,
+                                        const SemIR::Function& function,
+                                        SemIR::InstId import_ref) {
+  // Import if function is declared as extern and is a non-owning declaration
```

Comment:
```suggestion
// Imports the function if it's a non-owning declaration with the current file as owner.
static auto LoadImportForOwningFunction(Context& context,
                                        const SemIR::File& import_sem_ir,
                                        const SemIR::Function& function,
                                        SemIR::InstId import_ref) {
```

This is phrased a bit like a [function comment](https://google.github.io/styleguide/cppguide.html#Function_Comments), so suggesting to move out out of the function. "Imports" because of the implied "This function...", plus a period at the end. (and also, mentioning the "current file as owner" relationship)